### PR TITLE
v3 updates

### PR DESCRIPTION
**Pull request by robmcguinness**

### CODE REVIEW

File: scss/_cards.scss [hunk 1]
```diff
@@ -24,13 +24,15 @@
   }
 }
 
+$card-blockquote-color:rgba(0, 0, 0, 0.65);
```
Comment (TheSharpieOne) [inline scss/_cards.scss]: This should be in _variables.scss

File: scss/_cards.scss [hunk 2]
```diff
@@ -42,6 +44,19 @@
...
 }
 
+.card-warning, .card-default {
+  .card-blockquote {
+    color: $card-blockquote-color;
+  }
+}
+
+.card-block {
+  > .card-block {
```
Comment (TheSharpieOne) [inline scss/_cards.scss]: What's the use-case for this, `card-block`s shouldn't be nested like this. If the idea is to get padding on the left and right, `px-*` can be used.

File: docs/js/vendor.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
 import 'jquery';
-import 'chart.js';
 import 'expose?Tether!tether';
 import 'bootstrap/dist/js/bootstrap';
 import 'moment';
-import 'select2';
 import 'holderjs';
```
Comment (TheSharpieOne) [inline docs/js/vendor.js]: Isn't this docs only as well?

File: docs/content/javascript/select2.html [hunk 4]
```diff
@@ -1,12 +1,11 @@
 ---
 title: Select2
```
Comment (TheSharpieOne) [inline docs/content/javascript/select2.html]: We talk about select2 and call it select2 and even go as far as to show example jquery code using select2.... but when someone views the page, that person will not see it in action, just normal `select`s.

Comment (robmcguinness) [inline docs/content/javascript/select2.html]: - 71a665c removed select reference

File: scss/_bootstrap.scss [hunk 5]
```diff
@@ -3,7 +3,6 @@
 @import "variables";
 @import "~bootstrap/scss/bootstrap";
 @import "type";
-@import "external-libs";
 @import "animate";
 @import "app-icons";
 @import "awesome-bootstrap-checkbox";
```
Comment (robmcguinness) [inline scss/_bootstrap.scss]: Is there something in external libs that we need?

File: scss/_cards.scss (hunk 2 above)
Comment (robmcguinness) [inline scss/_cards.scss]: Hmm not sure and searching for selector rules that depend on hierarchy is a problem.  I'll remove and deal with this is other ways.  Might be old legacy junk.

File: scss/_cards.scss (hunk 1 above)
Comment (robmcguinness) [inline scss/_cards.scss]: - aa9b34f9 refactor to variables.scss done 😸

File: scss/_bootstrap.scss (hunk 5 above)
Comment (TheSharpieOne) [inline scss/_bootstrap.scss]: if we can do without it, I'd say go for it.

File: docs/layouts/partials/navbar-secondary-example.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
+<nav class="navbar navbar-default navbar-secondary navbar-toggleable-md mega-menu" role="navigation">
+  <div class="collapse navbar-collapse" id="availity-example-navbar-collapse-2">
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#availity-example-navbar-collapse-2">
+      &#9776;
```
Comment (TheSharpieOne) [inline docs/layouts/partials/navbar-secondary-example.html]: should this be `<span class="navbar-toggler-icon"></span>`

File: docs/content/components/navbar.html [hunk 7]
```diff
@@ -285,7 +285,7 @@ <h5 class="condensed">Pharmacy Authorization Approved</h5>
 <code class="language-markup">
 
 <nav class="navbar mega-menu navbar-default" role="navigation">
-    <button class="navbar-toggler hidden-md-up float-right" type="button" data-toggle="collapse" data-target="#availity-example-navbar-collapse-4">
+    <button class="navbar-toggler hidden-md-up navbar-toggler-right" type="button" data-toggle="collapse" data-target="#availity-example-navbar-collapse-4">
       &#9776;
```
Comment (TheSharpieOne) [inline docs/content/components/navbar.html]: this should be `<span class="navbar-toggler-icon"></span>`

File: docs/layouts/partials/navbar-secondary-example.html (hunk 6 above)
Comment (robmcguinness) [inline docs/layouts/partials/navbar-secondary-example.html]: fixed 7918b56da556aeb7f9192ef897e2970c597308ff

File: docs/content/components/navbar.html (hunk 7 above)
Comment (robmcguinness) [inline docs/content/components/navbar.html]: fixed 7918b56da556aeb7f9192ef897e2970c597308ff

**Pull request merged**